custom_components/google_home/const.py

```diff
@@ -57,6 +58,19 @@
 RECURRENCE = "recurrence"
 FIRE_TIME = "fire_time"
 ORIGINAL_DURATION = "original_duration"
+STATUS = "status"
+
+# TIMERS & ALARMS STATUS MAPPINGS/LABELS
+STATUS_TIMER_SET_INT = 1
+STATUS_TIMER_RINGING_INT = 3
+STATUS_ALARM_SET_INT = 1
+STATUS_ALARM_RINGING_INT = 2
+STATUS_ALARM_SNOOZED_INT = 3
+
+STATUS_NONE = "none"
+STATUS_SET = "set"
+STATUS_RINGING = "ringing"
+STATUS_SNOOZED = "snoozed"
```

Comment:
Do you guys think we should better go with a dictionary mapping here?
I am thinking of something like:
```python
STATUS_SET_LABEL = "set"
STATUS_SET_INT = 1
...
STATUS_MAPPING = {
  LABEL_ALARMS: {
   STATUS_SET_INT: STATUS_SET_LABEL,
    ... 
  },  
  LABEL_TIMERS: {
  ...
  },
}
```
Then we can reuse the same `STATUS_SET_LABEL` and `STATUS_SET_INT` for both timers and alarms and maintain a bit cleaner version of mapping. 

Then we can probs get rid of introduced `def map_timer_status()` when we need to match received status integer to a label in favour of:
```python
STATUS_MAPPING[LABEL_TIMER].get(status, None)
```

Reply:
I think it's better to use Enum here.

Reply:
Let's keep strings here but integer values inside `Enum` in `models.py` which maps these values to corresponding strings.

Reply:
@KapJI 
Thank you for suggesting using an Enum. I was not sure what you meant with keeping strings in `const.py` as I am not sure how you could use an Enum as name for a constant. I now removed status definitions completely from the constants and simply used `GoogleHomeAlarmStatus(status).name.lower()` to get a string value again.

Are you fine with this?

Reply:
Yes, that's great, thank you!